std/os/linux.zig

```diff
@@ -12,8 +12,76 @@ pub use switch (builtin.arch) {
     builtin.Arch.aarch64 => @import("linux/arm64.zig"),
     else => @compileError("unsupported arch"),
 };
+pub const syscalls = switch (builtin.arch) {
+    .x86_64 => @import("linux/syscalls_x86_64.zig"),
+    else => @compileError("NIY"),
```

Comment:
Nyi?

Reply:
Not Implemented Yet

Reply:
Normally the acronym is "not yet implemented"